sentry_sdk/integrations/rq.py

```diff
@@ -54,8 +64,8 @@ def setup_once():
         old_perform_job = Worker.perform_job
 
         @ensure_integration_enabled(RqIntegration, old_perform_job)
-        def sentry_patched_perform_job(self, job, *args, **kwargs):
-            # type: (Any, Job, *Queue, **Any) -> bool
+        def sentry_patched_perform_job(self, job, queue, *args, **kwargs):
```

Comment:
Checked that the signature of `perform_job` is stable in minimal supported version and in the current release. There are actually no `args` and `kwargs`, just `job` and `queue`, but we just for forwards compat.